ros_ws/src/autonomous/emergency_stop/launch/emergency_stop.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+<!-- Launches all necessary nodes for the emergency stop -->
+
+    <!-- emergency_stop node -->
+    <node
+      respawn="true"
+      pkg="emergency_stop"
+      type="emergency_stop"
+      name="emergency_stop"
+      output="screen" >
+    </node>
+
+</launch>
```

Comment:
This might be done in `gazebo.launch` instead in its own file. The screen output might not be necesary.

Reply:
Well it is like that in every other autonomous package as well, so why is this different to wallfollowing or q_learning?